Blockzilla/AppDelegate.swift

```diff
@@ -111,7 +114,25 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 rootViewController.present(firstRunViewController, animated: false, completion: nil)
             }
         }
-
+        
+        // Don't highlight whats new on a fresh install (prefIntroDone == 0 on a fresh install)
+        if prefIntroDone != 0 && UserDefaults.standard.string(forKey: AppDelegate.prefWhatsNewDone) != AppInfo.shortVersion {
+            
+            let counter = UserDefaults.standard.integer(forKey: AppDelegate.prefWhatsNewCounter)
+            switch counter {
+                case 0:
+                    // No counter set
+                    UserDefaults.standard.set(1, forKey: AppDelegate.prefWhatsNewCounter)
```

Comment:
Do we need this case if the default will give you `0 + 1`?

Reply:
good call!